spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,30 @@ class DeltaTable private[tables](
     toDataset(sparkSession, alterTableCmd)
   }
 
+  /**
+   * Modify the protocol to drop a supported feature. The operation always normalizes the
+   * resulting protocol. Protocol normalization is the process of converting a table features
+   * protocol to the weakest possible form. This primarily refers to converting a table features
+   * protocol to a legacy protocol. A Table Features protocol can be represented with the legacy
+   * representation only when the features set of the former exactly matches a legacy protocol.
+   * Normalization can also decrease the reader version of a table features protocol when it is
```

Comment:
"decrease the reader version" -> This should apply to writer version as well, so any version?